pkg/storage/swift/swift.go

```diff
@@ -328,12 +327,13 @@ func (d *driver) StorageChanged(cr *imageregistryv1.Config) bool {
 	return false
 }
 
-func generateContainerName(prefix string) string {
-	bytes := make([]byte, 16)
-	for i := 0; i < 16; i++ {
-		bytes[i] = byte(65 + rand.Intn(25)) // A=65 and Z=65+25
+func (d *driver) getClusterID() (string, error) {
```

Comment:
There is a util.GetInfrastructure(d.Listers) that you can use for this.

Reply:
https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/storage/s3/s3.go#L322

Reply:
Done!

Reply:
You should be able to dispense with the getClusterID function altogether, and use the util.GetInfrastructure where you are using the getClusterID.  You are basically wrapping a function that does what you want inside of another function that just returns it's value?

Reply:
you're right, this function doesn't make any sense

Reply:
You should be able to remove this method now